Foundation/NSDictionary.swift

```diff
@@ -39,6 +39,22 @@ open class NSDictionary : NSObject, NSCopying, NSMutableCopying, NSSecureCoding,
         
         return NSGeneratorEnumerator(_storage.keys.map { _SwiftValue.fetch(nonOptional: $0) }.makeIterator())
     }
+    @available(*, deprecated)
+    public convenience init?(contentsOfFile path: String) {
+        self.init(contentsOfURL: URL(fileURLWithPath: path))
+    }
+    
+    @available(*, deprecated)
+    public convenience init?(contentsOfURL url: URL) {
```

Comment:
The correct parameter name is `contentsOf` - see https://developer.apple.com/documentation/foundation/nsdictionary/1416069-init